packages/application-generic/src/usecases/get-decrypted-integrations/get-decrypted-integrations.usecase.ts

```diff
@@ -94,33 +48,16 @@ export class GetDecryptedIntegrations {
       ? [await this.integrationRepository.findOne(query)]
       : await this.integrationRepository.find(query);
 
-    const integrations = foundIntegrations
+    return foundIntegrations
       .filter((integration) => integration)
-      .map((integration: IntegrationEntity) => {
-        integration.credentials = decryptCredentials(integration.credentials);
-
-        return integration;
-      });
-
-    if (!isMultiProviderConfigurationEnabled) {
```

Comment:
@scopsy this is the case I was talking about where I assume we will enable this feature on relase of this branch